src/cmd/src/cli/export.rs

```diff
@@ -293,11 +293,14 @@ impl Export {
                 let (metric_physical_tables, remaining_tables) =
                     self.get_table_list(&catalog, &schema).await?;
                 let table_count = metric_physical_tables.len() + remaining_tables.len();
-                tokio::fs::create_dir_all(&self.output_dir)
+                let output_dir = Path::new(&self.output_dir)
+                    .join(&catalog)
+                    .join(format!("{schema}/"));
+                tokio::fs::create_dir_all(&output_dir)
                     .await
                     .context(FileIoSnafu)?;
                 let output_file =
-                    Path::new(&self.output_dir).join(format!("{catalog}-{schema}.sql"));
+                    Path::new(&output_dir).join(format!("{schema}_create_tables.sql"));
```

Comment:
Why not just put it into the `schema/` folder?

Reply:
Like `{schema}/create_tables.sql`

Reply:
Good point!

Reply:
It’s already in {schema}/; I just removed the file name prefix.